properties/prod/application.properties

```diff
@@ -100,6 +100,7 @@ athena.security.saml.attributes.email=email
 athena.security.saml.attributes.first_name=firstName
 athena.security.saml.attributes.last_name=lastName
 athena.security.saml.attributes.middle_name=middleName
+athena.security.saml.attributes.organization=organization
```

Comment:
It looks like there is a lot of duplication in properties. Do we have a base(common) `application.properties` that applies for all profiles? if so can you move all duplications there?

Reply:
no, there is not base properties